_posts/2020-05-26-how-to-pair-sucessfully.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "How to Pair Successfully"
+date: 2020-05-19 12:30:00
```

Comment:
@FionaDL I can publish this tomorrow. Please update the date here in the name of the file.